corehq/apps/users/models.py

```diff
@@ -1928,6 +1963,21 @@ def to_ota_restore_user(self, domain):
             self,
         )
 
+    def get_fixture_dataitems(self, domain):
+        return []
+
+    def get_groups(self, domain):
+        return []
+
+    def get_commtrack_location_id(self, domain):
+        return None
+
+    def get_owner_ids(self, domain):
+        return [self.user_id]
+
+    def get_call_center_indicators(self, domain):
+        return None
```

Comment:
kept things super simple for the initial web user restore, but we can get as fancy as we'd like depending how we feel web users should behave